src/plugins/workspace/public/components/workspace_overview/getting_start_card.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { EuiCard, EuiFlexGroup, EuiFlexItem, EuiText, EuiTextColor } from '@elastic/eui';
+import React from 'react';
+import { ApplicationStart, IBasePath } from 'opensearch-dashboards/public';
+import { formatUrlWithWorkspaceId } from '../../../../../core/public/utils';
+import { GetStartCard } from './types';
+
+export interface WorkspaceOverviewCardProps {
+  card: GetStartCard;
+  workspaceId: string;
+  basePath: IBasePath;
+  application: ApplicationStart;
+}
+
+export const WorkspaceOverviewCard = ({
+  card,
+  application,
+  workspaceId,
+  basePath,
+}: WorkspaceOverviewCardProps) => {
+  return (
+    <EuiCard
+      data-test-subj={card.featureName}
+      layout="vertical"
+      textAlign="left"
+      title={
+        <EuiText size="s">
+          <p>{card.featureDescription}</p>
+        </EuiText>
+      }
+      description={''}
+      footer={
+        <>
+          <EuiFlexGroup justifyContent="flexStart">
+            <EuiFlexItem grow={false}>
+              <EuiTextColor color="subdued">
+                <EuiText size="s">{'with ' + card.featureName}</EuiText>
+              </EuiTextColor>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </>
+      }
+      onClick={() => {
+        let url = card.link;
+        if (!url && card.appId) {
+          url = application.getUrlForApp(card.appId);
+        }
+
+        if (workspaceId && url) {
+          application.navigateToUrl(formatUrlWithWorkspaceId(url, workspaceId, basePath));
+        }
```

Comment:
```suggestion
        const url = card.link;
        if (!url && card.appId) {
          application.navigateToApp(card.appId);
          return ;
        }

        if (workspaceId && url) {
          application.navigateToUrl(formatUrlWithWorkspaceId(url, workspaceId, basePath));
        }
```
Better to use navigateToApp because that will use SPA routing to jump. And I do not think it is good to use a `link` to jump because it may refresh the page. Recommend using `defaultPath` to indicate the subApp path and keep consistent with the params for register Application. `link` can be used for external link maybe?

Reply:
`navigateToUrl` will try to parse the url first if it matches with any register app, it will use `navigateToApp` instead, otherwise use redirect
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/core/public/application/application_service.tsx#L300-L306

Reply:
Oh, I see, forget it.